packages/fxa-settings/src/components/InlineRecoveryKeySetupCreate/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InlineRecoveryKeySetupCreate renders as expected 1`] = `
+<button
+  class="cta-primary cta-xl flex justify-center items-center cta-primary cta-xl"
+  data-glean-id="inline_recovery_key_cta_submit"
+  type="submit"
+>
+  <span
+    data-testid="ftlmsg-mock"
+    id="inline-recovery-key-setup-start-button"
+  >
+    Create account recovery key
+  </span>
+</button>
+`;
+
+exports[`InlineRecoveryKeySetupCreate renders button with CMS passthrough 1`] = `
+<button
+  class="cta-primary-cms cta-xl flex justify-center items-center cta-primary cta-xl"
```

Comment:
`cta-primary cta-xl` is also unnecessary here + `cta-primary` and `cta-primary` shouldn't co-exist 🙃